test/handlers.jl

```diff
@@ -1,3 +1,13 @@
+using Test
```

Comment:
we should probably just require Compat.jl and then you can do `using Compat.Test` here.

Reply:
"[Remove Compat module; I hate Compat.jl.](https://github.com/JuliaWeb/HTTP.jl/commit/a89dddfc0e6eb4d6410090112438271d70277f60)"

Reply:
Yeah, that was back when it was used for a single compat thing. Now in this PR we have our own whole compat.jl. I think we use Compat for now, knowing that 0.7 will be out soon and we can drop it again then.

Reply:
I didn't notice compat.jl 🤔

Reply:
I've cleanup up compat.jl a little.
I have a very strong preference for avoiding Compat.jl if possible.
I've been burned in the past by hard to track down problems due to differing versions of Compat.jl on user machines and Compat.jl changing in subtle ways. I would much prefer to:
 - Suffer a few deprecation warnings and/or maintain a small local compat.jl.
 - Drop support for the previous version ASAP when a new version is released. Users of the previous version can still use the most recent tag that supported their version.